<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryWidget</class>
 <widget class="QWidget" name="QueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入要对比的内容：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_edt_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_mac">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>90</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>写MAC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_find">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>对比</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>74</y>
     <width>81</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择数据库：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_rx">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>90</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RX</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="m_radio_rf">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>90</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RF</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmb_table">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择表：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmb_field">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>200</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择字段：</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lbl_result">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>61</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>46</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
